src/raw/mod.rs

```diff
@@ -947,8 +953,9 @@ impl Deref for FstData {
 
     fn deref(&self) -> &[u8] {
         match *self {
-            FstData::Cow(ref v) => &**v,
+            FstData::Static(ref v) => v,
             FstData::Mmap(ref v) => unsafe { v.as_slice() },
+            FstData::SharedVector(ref v) => unsafe { v.as_slice() },
```

Comment:
Once you remove the `unsafe` annotation, you should be able to elide it here too.